### ISSUE TITLE
API-Create instability when uploading PDF

### ISSUE THREAD

**Issue by invokablegmbh** [bigbluebutton/bigbluebutton#10807]
**Describe the bug**
When doing CREATE api calls AND uploading a PDF (directly embedded in the body, NOT via URL) there is a chance that the API request will have an empty response. It seems like the bigger the PDF is and/or the higher the load on the server is, more API calls will fail.

**To Reproduce**
1. Setup a clean version of 2.2.28 server.
1. Do an API-Create call with a PDF embedded.
1. Execute this API call several times after one another.

**Expected behavior**
All API calls will succeed. The first call creates the conference, subsequent calls will return a duplication warning.

**Actual behavior**
For me, with a 20MB PDF embedded in the API call, the FIRST call succeeds. Subsequent calls almost all fail. There is a chance (maybe 1 out of 10 calls) that the api call goes through successfully.

**Additional information**
In the nginx log one can see "readv() failed (104: Connection reset by peer) while reading upstream". As nginx acts as a reverse proxy for the grails app, I did some debugging here. See first comment of this post.

**Comment by jarmediagmbh** [comment]
So I did some first debugging of this issue. I suspected it might have something to do with issue #9092 but as it turns out, this is not the case. So when I removed the "log" lines in code, nothing changed. The log was readable then, because no binary was dumped anymore. However, no errors were printed inside the grails app.
I then decided to search for this error somewhere on the nginx side. I increased upstream read/connect timeouts. No luck there either.
As we wanted to see what was happening on network level, we simply did a tcpdump on "lo" interface for port 8090. So we could see all traffic between nginx and the grails application. The interesting thing here is that we can see a totally correct conversation betweeen nginx and grails. Even we can see the http-packets containing the XML response of the grails application with a SUCCESS in it, although the error occured on that recorded tcpdump.
As we are PHP devs and are not that familiar with grails and the embedded tomcat server, we decided to stop debugging here.

Any ideas how to proceed?
